packages/lit-dev-content/site/docs/getting-started.md

```diff
@@ -45,6 +45,49 @@ import {LitElement, html} from 'lit';
 
 {% endswitchable-sample %}
 
+## Use bundles
+
+Lit is also available as pre-built, single-file bundles. The bundles are
```

Comment:
Would it make sense to enumerate some of the uses of this vs the "classic" npm approach?  e.g. rapid prototyping, cloud IDEs, etc just to help make it clear(er) why this option exists, but why the other is probably more favored / idiomatic?

Reply:
Thanks for taking a look! I've added some text that was suggested when I brought this up earlier today. We're trying to keep this section pretty short.